multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/model/ApplicationColor.java

```diff
@@ -12,4 +14,17 @@ public String asSuffix() {
         return "-" + super.toString().toLowerCase();
     }
 
+    public static String removeColor(String name) {
```

Comment:
Is this used anywhere?
I can't see it in the PR

Reply:
removed